backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/VdsManager.java

```diff
@@ -1293,11 +1333,40 @@ public void updateV2VJobInfos(List<V2VJobInfo> v2vJobInfos) {
         }
     }
 
+
     private void initAvailableCpus() {
         List<VdsCpuUnit> cpuTopology = cachedVds.getCpuTopology();
-        if (cpuTopology == null) {
+        if (cpuTopology == null || cachedVds.getVdsmCpusAffinity() == null) {
```

Comment:
can it be that the topology is returned but vdsm cpu affinity is null?

Reply:
not, it should be both or none.